config/environments/development.rb

```diff
@@ -64,4 +64,6 @@
   }
 
   config.project_name = ENV['HUNDRED_EYES_PROJECT_NAME'] || '100eyes'
+  # FIXME: this is needed to use localtunnel, the telegram widget only works over https
+  config.hosts << 'odd-shrimp-41.loca.lt'
```

Comment:
I have `APPLICATION_HOSTNAME` in #445 for that purpose, you can overwrite it with an environment variable.

Reply:
With [localtunnel](https://www.npmjs.com/package/localtunnel) you can request particular subdomains, so we could also register a dedicated domain here for development purposes.